testsuite/features/secondary/proxy_retail_pxeboot_and_mass_import.feature

```diff
@@ -12,10 +12,6 @@
 # * if there is no private network ($private_net is nil)
 # * if there is no PXE boot minion ($pxeboot_mac is nil)
 
-# TODO: This feature needs to be refactored for RBS containerized,
-#       as it does not support Salt formulas.
-#       Additionally, we have this bug that impedes to use mgr-bootstrap command https://bugzilla.suse.com/show_bug.cgi?id=1220864
```

Comment:
I still reading from the top to the bottom this PR, letting this message here, because for now I don't understand why we can remove this message, in particular, how we solved this issue: **_We have this bug that impedes to use mgr-bootstrap command https://bugzilla.suse.com/show_bug.cgi?id=1220864_**

Reply:
I am removing this message because this is precisely what this series of PRs is about: refactoring for Cobbler and Retail in a containerized setup.

In all logic, this removal would have to be done in next PR, not this one, but I was afraid of forgetting it, so I am doing it a bit in advance.

As for the bug, yes I know about it. If I am not wrong, it will only make the second half of this file (the mass import) fail. BTW it already fails in 4.3 for other reasons.

Reply:
Reverse YAGNI :)
Let's remove it only when is not anymore meaningful, together with the changes that invalidate it.

Reply:
I will re-add the remark about the bug, because my second PR will not make the issue go away.

Reply:
Thank you.

Reply:
Strictly speaking it is still meaningful in the context of this PR. I will re-add the whole comment.

If I forget to remove it in next PR it will be your fault :trollface: .

Reply:
Done